build.sbt

```diff
@@ -63,6 +63,7 @@ lazy val cluster = (project
     ConfigTools,
     Nel,
     scalax,
+    PlayJson,
```

Comment:
I don't really like adding play-json dependency on the library level

Reply:
What would you advise? To keep it as is and to make a permanent wrapper where it is needed?